src/nemo_run/test_utils.py

```diff
@@ -0,0 +1,93 @@
+# SPDX-FileCopyrightText: Copyright (c) 2024 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Comment:
This should be moved to the `test` folder.

Reply:
Doesn't work for the CLI, that's why I moved it inside the package.

Reply:
This file will be included in installation if its inside `src`, which is not ideal. Why doesn't it work with the cli? Can you explicitly load the file in the tests?